code/solutions/pseudo-parameters.yaml

```diff
@@ -0,0 +1,68 @@
+AWSTemplateFormatVersion : "2010-09-09"
+Description: >
+  This template will create a SSM Parameter and reference its ARN on an IAM policy
+  leveraging CloudFormation pseudo parameters
+
+Parameters:
+  S3BucketNamePrefix:
+    AllowedPattern: ^[0-9a-zA-Z]+([0-9a-zA-Z-]*[0-9a-zA-Z])*$
+    ConstraintDescription: Quick Start bucket name can include numbers, lowercase
```

Comment:
Please update the constraint description: remove description text that is out of context for this lab.